File: exercism_data/src/357.rb
class Phrase

  attr_reader :phrase

  def initialize(phrase)
    @phrase = phrase.downcase.gsub(",", " ").gsub(/[^a-z|\s|\d|\']/, "")
  end

  def word_count
    counts = {}
    phrase.split(" ").each do |word|
      if counts[word]
        counts[word] += 1
      else
        counts[word] =1
      end
    end
    counts
  end
end

File: assignments/python/exercism_wc/src/357.py
def word_count(phrase):
    word_count_dict={}
    word_split=phrase.split()
    for word in word_split:
        if word not in word_count_dict:
            word_count_dict[word]=1
        else:
            word_count_dict[word]+=1
    return word_count_dict